searchtool/server/views/layout.hbs

```diff
@@ -183,7 +183,8 @@
                           {{#if term}}
                             <button aria-expanded="false" aria-haspopup="true" data-toggle="dropdown" class="btn btn-default dropdown-toggle" id="btn-pgActions">Download <span class="caret"></span></button>
                             <ul aria-labelledby="btn-pgActions" class="dropdown-menu">
-                              <li role="presentation"><a target="_BLANK" href="/download?q={{term}}&dataset={{dataset}}&fromdate={{fromdate}}&todate={{todate}}&art_unit={{artunit}}&documentcode={{documentcode}}">Download CSV</a></li>
+                              <li role="presentation"><a target="_BLANK" href="/download?q={{term}}&dataset={{dataset}}&fromdate={{fromdate}}&todate={{todate}}&art_unit={{artunit}}&documentcode={{documentcode}}&textdata=yes">CSV with text</a></li>
```

Comment:
Thanks @bahadasx ! Can we change `yes` and `no` to `true` and `false`? Also is it on the user to know which one to pick or should we show/hide based on the number of results? If you don't have time, I'm happy to make these changes on your branch. Can you add me as a collaborator to your repo?